src/goto-instrument/cover.h

```diff
@@ -42,41 +42,46 @@ enum class coverage_criteriont
   PATH, MCDC, ASSERTION, COVER };
 
 bool consider_goals(
-  const goto_programt &goto_program,
-  coverage_goalst &goals);
+  const goto_programt &,
```

Comment:
I don't understand why these parameter names have been removed. Parameter names should be consistent between header and source files.